src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -318,16 +298,34 @@ object SbtWeb extends AutoPlugin {
     nodeModules := nodeModuleGenerators(_.join).map(_.flatten).value
   )
 
+  def webJarsPathPrefix: Def.Initialize[Task[String]] = Def.task {
+    path(s"$WEBJARS_PATH_PREFIX/${moduleName.value}/${version.value}/")
+  }
+
+  def syncExportedAssets(): Def.Initialize[Task[File]] = Def.task {
+    val syncTargetDir = classDirectory.value
+    val syncRequired = TrackLevel.intersection(trackInternalDependencies.value, exportToInternal.value) match {
+      case TrackLevel.TrackAlways =>
+        true
+      case TrackLevel.TrackIfMissing | TrackLevel.NoTracking =>
+        !(syncTargetDir / webJarsPathPrefix.value).exists()
+    }
+    if (syncRequired) {
+      state.value.log.info(s"Exporting Assets for ${moduleName.value}")
+      syncMappings(streams.value.cacheDirectory, exportedMappings.value, syncTargetDir)
```

Comment:
@eed3si9n By having made the task dynamic the multi-module test fails.

I have reproduced this by changing the tester project to reflect the multi-module test setup. When invoking `b/web-assets-test:assets`, the c.js file is no longer copied to the test folder. In sbt-web, compile scoped assets are copied along with their test assets given that this is what Node.js expects i.e. the source and test files must be relative to each other.

If I once more make the function a regular task then all appears well, notwithstanding your concern that various transpilers will kick in.

Any thoughts? Also @pvlugter - It isn't obvious to me how the compile assets end up with the test assets anyhow... do you recall how this works?